actions-unstable/secret/action.yml

```diff
@@ -0,0 +1,26 @@
+name: 'ggshield scan'
```

Comment:
name: 'ggshield secret scan'

Reply:
I just copied the actions/secret/action.yml here. I agree those needs to be updated, but this can be done in a separate issue.